aten/src/ATen/native/native_functions.yaml

```diff
@@ -3326,6 +3326,10 @@
 
 - func: pixel_unshuffle(Tensor self, int downscale_factor) -> Tensor
 
+- func: quantized_pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
```

Comment:
Why do you need a new op for this if it is only called from channel_shuffle? This could just be a different dispatch for channel_shuffle below no?

Reply:
@albanD Hi thanks! 
Updated with dispatch key of `QuantizedCPU` and `CompositeExplicitAutograd`.

If I use `CPU, CUDA` in the dispatch table, XLA test case would fail.

If I use `CompositeImplicitAutograd`, it gave the following error:
```
RuntimeError: 0INTERNAL ASSERT FAILED at "../aten/src/ATen/core/boxing/KernelFunction.cpp":23, please report a bug 
to PyTorch. aten::pixel_shuffle has kernels registered to both CompositeImplicitAutograd and a backend mapped to 
AutogradOther. This makes the backend kernel unreachable; the dispatcher will always prefer the 
CompositeImplicitAutograd lowering (see Note [Ambiguity in AutogradOther kernel]).
```